### PR TITLE
Bash scripting

### PR DESCRIPTION
**Pull request by drelliche**
As of right now the scripts referenced in this module are in a branch of the learning_bash repo: https://github.com/arcus/learning_bash/tree/scripts 

They will be merged once this module is approved. I have checked their addition doesn't interfere with the other bash modules, but this is also worth a double check.

### CODE REVIEW

**Comment by drelliche** [comment]
@franzenr You seem to be on a Bash roll, but let me know if you don't have bandwidth to QA this and I'll pass it to someone else.

I'm also hoping we will have an intern with a Windows laptop this winter to Windows-test all of these for us...

**Comment by rosemm** [comment]
I'll nab this one from Rose since she's out of the office for a bit!

File: bash_scripts/bash_scripts.md [hunk 1]
```diff
@@ -0,0 +1,642 @@
...
+
+author:   Elizabeth Drellich
+email:    drelliche@chop.edu
+version:  1.0.0
+module_template_version: 2.0.0
+language: en
+narrator: UK English Female
+title:  Bash: Reusable Scripts
+comment:  This module will teach you how to create and use simple bash scripts so to make repetitive tasks as simple as possible. If you have some experience with bash and want to learn how to save and reuse bash processes, this lesson is for you.
+estimated_time: 1 hour
```
Comment (rosemm) [inline bash_scripts/bash_scripts.md:11]: missing `long_description` here

**Review by rosemm — CHANGES_REQUESTED** [review]
Great module! I found a bunch of little things, but nothing major.

File: bash_scripts/bash_scripts.md [hunk 2]
```diff
@@ -0,0 +1,642 @@
...
+4. Place this new folder `learning_bash-main` somewhere you can easily find it. In the examples we will assume that `learning_bash-main` is in the Downloads directory, but you are welcome to move it somewhere else that is convenient for you to navigate to in your command line interface.
+
+## What is a script?
+
+A **script** is a small program that you can run on your computer.
+
+A **bash script** is a small program that automates bash commands and lets you run the same sequence of commands as many times as you want. It might be as simple as a program that takes a file you have downloaded and standardizes the file name, or as complicated as creating new files based on information stored in several other files in other locations or doing complex tasks based on a user's typed responses to questions.
+
+You might have heard the terms "script" and "bash scripting" and "shell script" used interchangeably. This is probably because the Bash language is most powerful when you use it to write short scripts that do exactly what you want. Once you get the hang of writing scripts, you can automate almost any work that you expect to need to use multiple times!
```
[the remark this answers is not in the document]
Comment (drelliche) [reply bash_scripts/bash_scripts.md:91]: Nope, this is definitely confusing and I will reword/rewrite it! Thanks for the heads up!

File: bash_scripts/bash_scripts.md (hunk 1 above)
Comment (drelliche) [reply bash_scripts/bash_scripts.md:12]: Oops!! added!

File: bash_scripts/bash_scripts.md [hunk 3]
```diff
@@ -0,0 +1,642 @@
...
+comment:  This module will teach you how to create and use simple bash scripts so to make repetitive tasks as simple as possible. If you have some experience with bash and want to learn how to save and reuse bash processes, this lesson is for you.
+estimated_time: 1 hour
+
+@learning_objectives
+
+After completion of this module, learners will be able to:
+
+- Identify the structure of a Bash script
+- Run existing Bash scripts
+- Write simple Bash scripts
```
[the remark this answers is not in the document]
Comment (drelliche) [reply bash_scripts/bash_scripts.md:20]: Hmmmm... I went through and capitalized it everywhere it is in a sentence, to see how jarring that would be. Would you take a look at see if "technically correct" is in fact the best form of correct in this case? If not, I can easily change them all back to lowercase unless they start a sentence.

File: bash_scripts/bash_scripts.md [hunk 4]
```diff
@@ -0,0 +1,642 @@
...
+**Pre-requisites**
+
+Learners should be familiar with using a bash shell to navigate a directory system. Learners will get the most out of this lesson if they can also create directories and files, write text to files, and read files from their bash shell command line interface.
+
+Bash commands that will be used without explanation include:
+- `>` and `>>`
+- `echo`
+- `grep`
+- `wc`
+
```
[the remark this answers is not in the document]
Comment (drelliche) [reply bash_scripts/bash_scripts.md:54]: added!

File: bash_scripts/bash_scripts.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (drelliche) [reply bash_scripts/bash_scripts.md]: I've removed some of this paragraph and folded one sentence into the highlight box after it, which is now a cool-fact box. 

Let me know whether it makes any sense now.

**Review by rosemm — COMMENTED** [review]
Ah, yes, thank you! I think that's much clearer.

File: bash_scripts/bash_scripts.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (rosemm) [reply bash_scripts/bash_scripts.md:20]: I don't find it jarring, actually. So I think my inclination would be to capitalize. Although now that I'm looking for it, I notice that we have inconsistent capitalization of "Bash" in the other modules (sigh), e.g. [Bash 101](https://liascript.github.io/course/?https://raw.githubusercontent.com/arcus/education_modules/main/bash_command_line_101/bash_command_line_101.md#1). 
But I vote to deal with that later, and for this module I say you're good.

File: bash_scripts/bash_scripts.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (rosemm) [reply bash_scripts/bash_scripts.md]: Love it :)

**Review by rosemm — APPROVED** [review]
This looks good to me!

**Pull request merged**